### ISSUE TITLE
Wrong desktop environment - Use the Gnome

### ISSUE THREAD

**Issue by max-rosenfors** [basebelong/ubuntu-postinstall-cinnamon#1]
Reading in the [postinstall-script](https://github.com/basebelong/ubuntu-postinstall-cinnamon) I can see that you have chosen to use the cinnamon desktop environment, instead of the Gnome that comes with Ubuntu.

This is a mistake because of reasons and various compatibility features over there and so.

I recommend rethinking many of these decisions while this project is still in early development. While there is still time. After feedback in this thread I make a pull request with the necessary changes.

**Comment by basebelong** [comment]
I agree. As GNOME is default in both RHEL and Fedora, everyone should use it. Canonicals theming is pushing it.

In fact, Alma Linux should be used. It is RHEL but with a merrier wallpaper. Theming is important, but should stay at the wallpaper level. GNOME GTK is glorious as is. Failing switching to Alma Linux - Oracle has a bad ass Tux in body armour as logotype. Only rivaled by Temple OS. But that is not Linux. Hanna Montana Linux could also be a contender.
